sdk/identity/identity/src/credentials/environmentCredential.ts

```diff
@@ -47,10 +47,9 @@ export class EnvironmentCredential implements TokenCredential {
     | ClientCertificateCredential
     | UsernamePasswordCredential = undefined;
   /**
-   * Creates an instance of the EnvironmentCredential class and reads
-   * client secret details from environment variables.  If the expected
-   * environment variables are not found at this time, the getToken method
-   * will return null when invoked.
+   * Creates an instance of the EnvironmentCredential class and reads client secret details from environment variables.
```

Comment:
Nit: certificates as well as username/password can also be used, maybe they should be mentioned as well (currently it just says "client secret details")?